src/repositories/projectPowerViewRepository.test.ts

```diff
@@ -470,7 +518,7 @@ function getTopPowerRankTestCases() {
     await setPowerRound(roundNumber);
     await refreshProjectPowerView();
 
-    const topPowerRank = await getTopPowerRank();
+    const topPowerRank = await getBottomRank();
```

Comment:
rename variable. I understand now better what you meant, I was the one that named it wrongly.